src/lib/userStorage/UserStorageClass.js

```diff
@@ -1009,7 +971,7 @@ export class UserStorage {
   }
 
   // eslint-disable-next-line require-await
-  async getPublicProfile(key: string, string: string): Promise<any> {
+  async getPublicProfile(key: string, string?: string): Promise<any> {
```

Comment:
this is mandatory it is not optional

Reply:
you didt merge with latest master

Reply:
This is after merging getUserProfile with get Public Profile, in getUserProfile we were only passing the value, and based on thi s value we are checking what kind of field it is.